content/desktop/contributing-and-collaborating-using-github-desktop/stashing-changes.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Stashing changes
+intro: You can stash changes to temporarily save them without committing them to a branch.
+versions:
+  free-pro-team: '*'
+---
+
+### About stashed changes
+You can view and make commits to any of your repository's branches. If you have uncommitted, saved changes, you'll need to decide what to do with your changes before you can switch branches. You can commit your changes on the current branch, stash your changes on the current branch, or bring the changes to your new branch. If you want to commit your changes on the current branch, follow the steps in "[Committing and reviewing changes to your project](/desktop/contributing-to-projects/committing-and-reviewing-changes-to-your-project)" before switching branches.
+
+You can stash all uncommitted changes to temporarily save the changes without committing them, and then reapply the changes later. You can only stash one set of changes at a time with {% data variables.product.prodname_desktop %}. After you stash changes on a branch, you can safely change branches or make other changes to your current branch.
```

Comment:
We often talk about committing as applying changes to a repository, so let's avoid using it to mean something else here. Desktop itself uses "restore" so let's use that for consistency.

The first paragraph should go back to the "Managing branches" article, IMO, under the "Switching between branches" heading.

```suggestion
To apply your changes to your repository, you must save the files and then commit the changes to a branch. If you have saved changes that you're not ready to commit yet, you can stash the changes for later. Stashing temporarily removes the changes from the files, but allows you to restore the changes later. You can only stash one set of changes at a time with {% data variables.product.prodname_desktop %}. After you stash changes on a branch, you can safely change branches or make other changes to your current branch.
```

Reply:
Let's also clarify that you can only stash ALL the currently saved changes, and that it's not possible to stash a subset of changes. My first instinct was to test whether the latter was possible, as many things you can do to all changes (e.g., discard them), you can also do to individual files or even lines.